core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -573,23 +561,27 @@ public void resetOffsetsTo(final Consumer<byte[], byte[]> client,
         return topicPartitionAndOffset;
     }
 
-    private Map<TopicPartition, Long> checkOffsetRange(final Map<TopicPartition, Long> inputTopicPartitionsAndOffset,
+    private Map<TopicPartition, Long> checkOffsetRange(
+                                                       final Map<TopicPartition, Long> inputTopicPartitionsAndOffset,
                                                        final Map<TopicPartition, Long> beginningOffsets,
                                                        final Map<TopicPartition, Long> endOffsets) {
```

Comment:
I think that does not conform to our code style. I think it should look like:
```suggestion
    private Map<TopicPartition, Long> checkOffsetRange(final Map<TopicPartition, Long> inputTopicPartitionsAndOffset,
                                                       final Map<TopicPartition, Long> beginningOffsets,
                                                       final Map<TopicPartition, Long> endOffsets) {
```